### DISCUSSION TITLE
[Feature] Select date/time format separately from locale

### DISCUSSION THREAD

**Discussion by marsmathis** [immich-app/immich#14290] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I’d like to be able to select the ISO 8601 date format without having to resort to selecting Swedish as my locale (which sets the month short names to the Swedish abbreviations). Maybe call it ISO 8601 (International) or something.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by seekified** [comment]
As a start, could we maybe have an override for showing timestamps in a 24-hour format? I want my apps in English but as a European I don't use AM/PM. Normally my workaround for this is selecting UK English but that doesn't seem to be an option in Immich.

It's a tiny complaint in the scheme of things, but still 😅
